### PR TITLE
Fix id broken link.

### PR DESCRIPTION
**Pull request by irvifa**
There's various broken link based on [our ci](https://app.circleci.com/pipelines/github/jk8s/sig-docs-id/31/workflows/3d7383cb-788d-42d9-9469-13cae3fe0b74/jobs/49)

### CODE REVIEW

**Comment by irvifa** [comment]
Added temporary gcal event fix https://github.com/kubernetes/website/pull/22223

**Comment by sftim** [comment]
@irvifa, please rebase this pull request against _master_ to bring in a work-around for issue #22222.

**Comment by irvifa** [comment]
Hi @sftim it's already rebased from master by  the way, after it's merged..

**Comment by sftim** [comment]
Ah, cool, this doesn't need any rework by the look of it. Sorry about the noise.

**Comment by sftim** [comment]
(Aside) I think it's better not to mix whitespace tidying with other changes.

Ideally, either:
 - use a separate commit for the whitespace tidying
 - leave whitespace untouched even when it's not quite right

It's a minor thing though, this feedback doesn't mean “don't merge this”.

File: content/id/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -984,14 +984,14 @@ sehingga Deployment tidak akan dapat dikembalikan.
...
+Jika kamu ingin merilis ke sebagian pengguna atau server menggunakan Deployment,
 kamu dapat membuat beberapa Deployment, satu tiap rilis, dengan mengikuti pola canary yang didesripsikan pada
 [mengelola sumber daya](/id/docs/concepts/cluster-administration/manage-deployment/#deploy-dengan-canary).
 
 ## Menulis Spesifikasi Deployment
 
 Sebagaimana konfigurasi Kubernetes lainnya, Deployment memerlukan kolom `apiVersion`, `kind`, dan `metadata`.
-Untuk informasi umum tentang penggunaan berkas konfigurasi, lihat dokumen [deploy aplikasi](/id/docs/tutorials/stateless-application/run-stateless-application-deployment/),
+Untuk informasi umum tentang penggunaan berkas konfigurasi, lihat dokumen [deploy aplikasi](/docs/tutorials/stateless-application/run-stateless-application-deployment/),
 mengatur kontainer, dan [memakai kubectl untuk mengatur sumber daya](/id/docs/concepts/overview/working-with-objects/object-management/).
```
Comment (wahyuoi) [inline content/id/docs/concepts/workloads/controllers/deployment.md:995]: if someone translates the run-stateless-application-deployment page, we need to update this line to add /id, right?
in this case, I think we need to check for every translated page, whether or not has impact to other pages
but it's only workaround while waiting for this issue https://github.com/kubernetes/website/issues/18403

File: content/id/docs/concepts/workloads/controllers/deployment.md [hunk 2]
```diff
@@ -1047,65 +1047,65 @@ Semua Pod yang ada dimatikan sebelum yang baru dibuat ketika nilai `.spec.strate
 
 #### Membarui Deployment secara Bergulir
 
-Deployment membarui Pod secara [bergulir](/id/docs/tasks/run-application/rolling-update-replication-controller/)
```
Comment (wahyuoi) [inline content/id/docs/concepts/workloads/controllers/deployment.md:1050]: I think this pointing to a different link? @irvifa 

But anyway, on /en page, there is no link anymore for this sentence.
I suggest removing the link to match with the /en

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#rolling-back-a-deployment

Comment (irvifa) [reply content/id/docs/concepts/workloads/controllers/deployment.md:1050]: Unfortunately I can't find a new link for this, as for your suggestion I already included it on my latest commit.

File: content/id/docs/concepts/workloads/controllers/deployment.md (hunk 1 above)
Comment (wahyuoi) [reply content/id/docs/concepts/workloads/controllers/deployment.md]: hi @irvifa why did you remove this changes?

I think removing /id/ from `deploy aplikasi` is fine.
I'll create a checker to accommodate the workaround on my previous comment :crossed_fingers:

File: content/id/docs/concepts/workloads/controllers/deployment.md (hunk 2 above)
Comment (wahyuoi) [reply content/id/docs/concepts/workloads/controllers/deployment.md:1050]: I think there is no changes for this line
will put the code suggestion

File: content/id/docs/concepts/workloads/controllers/deployment.md [hunk 3]
```diff
@@ -1047,65 +1047,65 @@ Semua Pod yang ada dimatikan sebelum yang baru dibuat ketika nilai `.spec.strate
...
+Deployment membarui Pod secara [bergulir](/docs/tutorials/stateless-application/)
```
Comment (wahyuoi) [inline content/id/docs/concepts/workloads/controllers/deployment.md:1050]:
```suggestion
Deployment membarui Pod secara bergulir
```

**Comment by irvifa** [comment]
@wahyuoi I already rebased and squashed the commits.

**Pull request merged**